third_party/cabal2bazel/Readme.md

```diff
@@ -0,0 +1,33 @@
+Hazel uses some code originally hosted at
+[google/cabal2bazel](https://github.com/google/cabal2bazel).
+The current version of that code was taken from [this commit](https://github.com/google/cabal2bazel/commit/c208bf179c8e820b0733a4472a5825c4f86e8e60).
+
+Some of that code is used nearly unchanged; other parts (in particular,
+`cabal_package.bzl` are heavily modified.  Here is a (nonexhaustive) list of
+some key differences:
+
+#### Different semantics of `rules_haskell`
+- `hsc` files can be passed directly to build rules.
+- All source files need to be given a common `src_strip_prefix`; solve this
+  by symlinking them all into a common location.
```

Comment:
I know you probably don't care at the moment. Is there a generic solution that bazel espouses for symlinking on Win?

Reply:
I don't think so; there isn't really a generic "official" solution for symlinking on *nix systems either (see below).